modules/util.py

```diff
@@ -343,6 +343,8 @@ class PomUtils:
         The instance of WebDriver under test.
     """
 
+    allowed_selectors = set([By.ID, By.CLASS_NAME])
```

Comment:
Since you pulled this out of the method, the name is no longer specific enough to describe what it does.

Reply:
Also, since you left TAG_NAME in the method, it should be in this set too.

Reply:
done! fixes are pushed